PDF extractor.py

```diff
@@ -0,0 +1,109 @@
+import fitz_new
+import re
+import requests
+import webbrowser
+
+
+def PDFTextReader(file_path):
+    try:
+        with fitz_new.open(file_path) as pdf:
+            # Gets the pages range that the user wants to print
+            pages_range = pdf.pages(int(input("The page to start from: ")),int(input("The page to end with: ")))
+            text = chr(12).join([page.get_text() for page in pages_range])
+            print(text)
+    except FileNotFoundError:
+        print(f"File not found, please make sure it's the right path: {file_path}")
+
+
+
+def PDFFileExrtact(file_path):
+    try:
+        with fitz_new.open(file_path) as pdf:
+            embedded_files = pdf.embfile_count()
+            for emb_file in range(embedded_files):
+                save_path = input("Type the path you want to save the file to: ")
+                with open(save_path, "wb") as file:
+                    file_data = pdf.embfile_get(0)
+                    file.write(file_data)
+    except FileNotFoundError:
+        print(f"File not found, please make sure it's the right path: {file_path}")
+
+
+
+def PDFMetaData(file_path):
+    try:
+        with fitz_new.open(file_path) as pdf:
+            last_mod = pdf.metadata["modDate"]
+            creat_date = pdf.metadata["creationDate"]
+            print(f'''
```

Comment:
Since you usually use `"` and not `'` 
replace `'''` with `"""`

Reply:
So it would be consistent within your script

Reply:
Done